android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -183,7 +198,7 @@ public ReactExoplayerView(ThemedReactContext context) {
         themedReactContext.addLifecycleEventListener(this);
         audioBecomingNoisyReceiver = new AudioBecomingNoisyReceiver(themedReactContext);
 
-        initializePlayer();
+        // initializePlayer();
```

Comment:
Should this really be commented? With the current state of master this breaks the use of controls.

Reply:
It is being called on `onAttachedToWindow`.
By controls what do you mean? Player UI?

Reply:
By controls I'm referring to the addition of controls for exoplayer in #1414. Your solution without passing the prop controls works brilliantly. Although when passing controls prop it crashes with a NullPointerException due to the player not being initialized yet.

Reply:
Thank you so much @keisnet. I will take a look, thanks for reporting

Reply:
@keisnet fixed. Can you check on https://github.com/react-native-community/react-native-video/pull/1445/commits/decb56e6899c35975e474f8f160cdae2c43e9502 ?

Reply:
@danielmarino24i I'll try to give it a check today.

Reply:
Any feedback @keisnet